frog/server/user_accounts.js

```diff
@@ -13,7 +17,7 @@ const doLogin = (user, self) => {
       return Accounts._loginUser(self, alreadyUser._id);
     }
   }
-  const userServiceData = { id: user || uuid() };
+  const userServiceData = { id: user || uniqueNamesGenerator(usernameConfig) };
```

Comment:
can we make the username capital first letters, like Happy Panda instead of happy panda?

Reply:
Made the changes